coding-standards/html-css/README.md

```diff
@@ -1,40 +1,93 @@
 # HTML/CSS Coding Standards
-Author: Mattijs Bliek
+Authors: Michèle van den Aardweg, Koen Schaft, Mattijs Bliek
```

Comment:
Adding links would be nice. Then I can easily look for information about these inspiring people.

Reply:
Added some Twitter links 👍